src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ca/AssessmentService.java

```diff
@@ -39,6 +39,7 @@ public class AssessmentService {
       "Dimension name provided [%s] does not match actual dimension name for id [%d] - expected [%s]";
   private static final String ERR_MSG_FMT_DIMENSION_ID_NOT_MATCH_ID =
       "Dimension-id [%d] does not match dimension-id [%d]";
+  private static final String ERR_MSG_FMT_DIMENSION_INVALID_ID = "Invalid Dimension-id [%d]";
```

Comment:
It is the criterion value so something like this might be more accurate:
```
ERR_MSG_FMT_INVALID_CRITERION_INTEGER = "Invalid criterion value [%d] - must be positive integer";